quirks/password-rules.json

```diff
@@ -35,6 +35,9 @@
     "brighthorizons.com": {
         "password-rules": "minlength: 8; maxlength: 16;"
     },
+    "capitalone.com": {
+        "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; allowed: [-_./\\@$*&!#];"
```

Comment:
I’m sad that you had to escape this. Maybe we should explore data formats that don’t involve having to escape the password rules string.